indico/modules/categories/controllers/management.py

```diff
@@ -105,6 +106,8 @@ class RHManageCategorySettings(RHManageCategoryBase):
     def _process(self):
         additional_keys = ('google_wallet_application_credentials', 'google_wallet_issuer_name',
                            'google_wallet_issuer_id')
+        if not config.ENABLE_GOOGLE_WALLET:
+            self.category.google_wallet_settings = {}
```

Comment:
I don't think those settings should be wiped out when the setting is disabled. Just leave them as-is (ie don't touch them).

Reply:
If I keep em, if you switch to ENABLE_GOOGLE_WALLET = False, existing Google Wallet links will still be there.

Reply:
Then that page should check the setting as well...

Reply:
Ok, I'll add the check to the RegistrationForm class, is_google_wallet_configured property.